File: P120876.py
def solution(lines):
    answer = 0
    grid = [0] * 201
    for line in lines:
        a, b = line[0], line[1]
        for i in range(a, b):
            grid[i+100] += 1
    for i in range(201):
        if grid[i] >= 2:
            answer += 1
    return answer

File: P120876.java
public class P120876 {
    public int solution(int[][] lines) {
        int answer = 0;
        int[] grid = new int[201];
        for (int[] line:lines) {
            int a = line[0];
            int b = line[1];
            for (int i=a; i<b; i++) grid[i+100] += 1;
        }
        for (int i=0; i<201; i++) {
            if (grid[i] >= 2) answer += 1;
        }
        return answer;
    }
}
